src/index.js

```diff
@@ -0,0 +1,152 @@
+import {
+  task, todoTask, saveTaskId, saveTask,
+} from './savetask.js';
+import check from './checkbox.js';
+
+const todoList = document.querySelector('#todo');
+const newTask = document.getElementById('new-task');
+let moreBtn;
```

Comment:
- [ ] Kindly consider using `const` instead of `let` (according to ES6) when declaring variables that won't be reassigned. For example, moreBtn can be declared with const instead of let.
```suggestion
const moreBtn = document.querySelectorAll('.drop-btn');
```